packages/gluestick/src/config/compileWebpackConfig.js

```diff
@@ -134,6 +136,18 @@ module.exports = (
         : prev;
     }, serverEnvConfig);
 
+  const pathToWebpackConfigHooks: string = path.join(process.cwd(), 'src', 'webpack.hooks');
+  const { webpackClientConfig, webpackServerConfig }: Hooks =
+    require(pathToWebpackConfigHooks).default;
```

Comment:
what if it doesn't have that file?? I think we should log warn but let it go further